model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerPod.java

```diff
@@ -109,6 +190,51 @@ void fillInFrom(ServerPod serverPod1) {
     readinessProbeTuning.copyValues(serverPod1.readinessProbeTuning);
     for (V1Volume var : serverPod1.getAdditionalVolumes()) addIfMissing(var);
     for (V1VolumeMount var : serverPod1.getAdditionalVolumeMounts()) addIfMissing(var);
+    serverPod1.nodeSelectorMap.forEach(nodeSelectorMap::putIfAbsent);
+    copyValues(resourceRequirements, serverPod1.resourceRequirements);
+    copyValues(podSecurityContext, serverPod1.podSecurityContext);
+    copyValues(containerSecurityContext, serverPod1.containerSecurityContext);
+  }
+
+  private void copyValues(
```

Comment:
I don't understand this either...  Why not just use the object from the spec?  This will again cause us pain if new fields are added.

Reply:
This is for granularity, with this implementation customers can configure the securityContext at domain or cluster level and override just some values for a particular server. If I remove this logic they would have to define the entire securityContext if they want to modify a single value from the domain or cluster.

Reply:
Okay, I'll let this go.  I've looked at the K8s history and these schemas have been stable for a long time.